curriculum/challenges/english/20-upcoming-python/learn-classes-and-objects-by-building-a-sudoku-solver/6568c0013b3b62d7617518c7.md

```diff
@@ -0,0 +1,77 @@
+---
+id: 6568c0013b3b62d7617518c7
+title: Step 37
+challengeType: 20
+dashedName: step-37
+---
+
+# --description--
+
```

Comment:
My suggestion is to ask the camper to create the following:
```py
if num in self.board[row]:
   return False
else:
   return True
```
And then refactor the code to obtain `return num not in self.board[row]`.